src/rust/engine/options/Cargo.toml

```diff
@@ -18,7 +18,8 @@ whoami = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
 serde_yaml = { workspace = true }
-parking_lot = "0.12.3"
+parking_lot = { workspace = true }
```

Comment:
This was an oversight, it should always have been taken from the workspace version.